apim/3.x/templates/gateway/gateway-service-account.yaml

```diff
@@ -19,6 +19,13 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: Role
 metadata:
   name: gravitee-gateway-controller
+  labels:
```

Comment:
There seems to be a problem with that change. When we try to deploy this we get the  error converting YAML to JSON: yaml: line 30: did not find expected key

Probably the spaces need to be deleted

https://stackoverflow.com/questions/58861962/error-converting-yaml-to-json-did-not-find-expected-key/58874298#58874298

Reply:
Are you using your own values.yaml @mteubner 

I'm not able to reproduce it from my side.

Reply:
We are using flux to deploy the current gravitee chart.

I will try to reproduce it with a plain only gravitee installation and create an issue with reproducing steps.

Just wanted to highlight first, that all other files don't have spaces in between the brackets { {

But gateway-service-account.yaml does. Anyways i will file an issue, when I have easy reproduction.

Reply:
Version 3.1.32 is working, all later fail.

Reply:
Indeed, good catch.

I'm wondering why it's working well with helm unittests and why we didn't get such issue before....

Reply:
Ok, I succeeded to reproduce the issue, looking at fixing it now

Reply:
cool, shall I still add an issue, or is it in good hands now :)

Reply:
I've just pushed Helm chart 3.1.47 for fixing this issue @mteubner

Reply:
hey @brasseld, i'm a team-member of @mteubner and just tested the new version of the helm chart 3.1.47 - unfortunately there seem to be another issue: 

```
# flux source defintion 
      Source Ref:
        Kind:       HelmRepository
        Name:       graviteeio
        Namespace:  env-common
      Version:      3.1.47
```

here is the error message from the helm controller: 

```
Status:
  Conditions:
    Last Transition Time:  2022-06-15T11:42:04Z
    Message:               Helm install failed: error while running post render on files: map[string]interface {}(nil): yaml: unmarshal errors:
  line 55: mapping key "serviceAccountName" already defined at line 36
    Reason:                InstallFailed
    Status:                False
    Type:                  Ready
    Last Transition Time:  2022-06-15T11:42:04Z
    Message:               Helm install failed: error while running post render on files: map[string]interface {}(nil): yaml: unmarshal errors:
  line 55: mapping key "serviceAccountName" already defined at line 36
  ```
  
  our values do look like the following: 
  
  ```
      es: 
      endpoints: 
        - http://elasticsearch-master.env-common.svc.cluster.local:9200
    ingress: &gravitee_ingress
      annotations:
        kubernetes.io/ingress.class: nginx-external
        cert-manager.io/cluster-issuer: letsencrypt-dev
      hosts: 
        - gravitee.${baseFQDNDev}
      tls:
        - hosts:
            - gravitee.${baseFQDNDev}
          secretName: gravitee.${baseFQDNDev}
    ui:
      ingress: *gravitee_ingress
    portal:
      ingress: *gravitee_ingress
    api:
      ingress:
        portal: *gravitee_ingress
        management: *gravitee_ingress
      livenessProbe:
        initialDelaySeconds: 240
      readinessProbe:
        initialDelaySeconds: 240
      additionalPlugins:
        - https://xxxxx.xxxxxxxx/plugins/gravitee-policy-passport-header-1.0.0-SNAPSHOT.zip
    gateway:
      ingress: 
        annotations:
          kubernetes.io/ingress.class: nginx-external
          cert-manager.io/cluster-issuer: letsencrypt-dev
          nginx.ingress.kubernetes.io/whitelist-source-range: 0.0.0.0/0 
        hosts: 
          - gateway.${baseFQDNDev}
        path: /
        tls:
          - hosts:
              - gateway.${baseFQDNDev}
            secretName: gateway.${baseFQDNDev}
      services:
        kubeController:
          enabled: true
          serviceAccount: "gravitee-gateway-sa"
   ```
   
   is it a mistake we do? or do we spotted another issue here?

Reply:
Are you sure you need this part ?
```
```

Reply:
> Are you sure you need this part ?
> 
> ```
>     services:
>       kubeController:
>         enabled: true
>         serviceAccount: "gravitee-gateway-sa"
> ```
> 
> ?

im not. i was wondering as well if this could be the issue. we will test and let you know. sorry in ahead if my last comment was causing confusion :)

Reply:
> Are you sure you need this part ?
> 
> ```
>     services:
>       kubeController:
>         enabled: true
>         serviceAccount: "gravitee-gateway-sa"
> ```
> 
> ?

removed and now it works! sorry... and thanks for your quick fix with the labels issue!

Reply:
47 does not fail on deployment with flux. Thx

Reply:
Thanks guys 👍